repository.rosinstall

```diff
@@ -22,3 +22,8 @@
     uri: https://github.com/shadow-robot/common_resources.git
     version: 'noetic-devel'
     local-name: common_resources
+
+- git:
+    uri: https://github.com/shadow-robot/sr_hand_config.git
```

Comment:
If we are going to make this repository private, we shouldn't create a dependency on it.
Is it possible to have a public branch of a repo, but then keep some others private?